lib/complex.ex

```diff
@@ -291,13 +367,16 @@ defmodule Complex do
   [new/2](#new/2), [abs/1](#abs/1)
 
   #### Examples
-      iex> Complex.abs_squared( Complex.fromPolar(1, :math.pi/2) )
+      iex> Complex.abs_squared( Complex.from_polar(1, :math.pi/2) )
       1.0
 
-      iex> Complex.abs_squared( Complex.fromPolar(2, :math.pi/2) )
+      iex> Complex.abs_squared( Complex.from_polar(2, :math.pi/2) )
```

Comment:
We should also remove the space before and after parens, but it can also be done later. :)